cmd/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/gojp/goreportcard/check"
+	"github.com/gojp/goreportcard/handlers"
+)
+
+var allScores []handlers.Score
```

Comment:
i found it kind of weird calling this `handlers.Score`, do we want to add another package for the Grade and Score types maybe?